plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -212,7 +212,13 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
                EOS_ASSERT(false, plugin_exception, "");
             }
 #endif
-         }), "Override default WASM runtime")
+         }), "Override default WASM runtime(\"eos-vm\", \"eos-vm-jit\", \"eos-vm-oc\").\n"
```

Comment:
I thought we were going to set the default `eos-vm` here so it shows up as default. If we don't set the default here we should at least indicate what it is.

Reply:
If I add default_value(eosio::chain::wasm_interface::vm_type::eos_vm, "eos-vm") call, 3 integration tests failed. I think It is not worth to add default_value for now.
The following tests FAILED:
	 87 - db_modes_test (Failed)
	 88 - release-build-test (Failed)
	 89 - validate-reflection-test (Failed)

Reply:
It is fine to just indicate in the description that `eos-vm-jit` is the default. However, please investigate why these tests failed.

Reply:
Looking at why default_value() call not working, I found a bug in chain_plugin constrtuctor:
app().regtister_config_type<eosio::chain::wasm_interface::vm_type>() was not called. As a result wasm-runtime can't be processed. An error message appeared in nodeos.log
"APPBASE: Developer -- the type N5eosio5chain14wasm_interface7vm_typeE is not registered with appbase,  add a register_config_type<>() in your plugin's ctor."

After I fixed this, the default_value() call works and integration test passed, too.